### ISSUE TITLE
Officially support React

### ISSUE THREAD

**Issue by jennmueng** [getsentry/sentry-capacitor#40]
Add an example app, and fully test all features.

**Comment by davidemerritt** [comment]
Is this on the roadmap, does it only work with angular right now?

**Comment by jennmueng** [comment]
@davidemerritt Yep this is a roadmap item. It _officially_ only works with angular, however in theory React should already work just fine and you can pass `@sentry/react` as the second parameter to this init call not unlike the angular SDK. If you run into any issues you can just [initialize the standalone Sentry Capacitor SDK](https://docs.sentry.io/platforms/javascript/guides/capacitor/#install)(Use the "Other Frameworks" tab).

**Comment by davidemerritt** [comment]
I have it working as you described but wondering what will break... Thanks for the update.

**Comment by piotr-cz** [comment]
Works fine in production

```tsx
import * as Sentry from '@sentry/capacitor'
import { init as sentryReactInit } from '@sentry/react'

Sentry.init({
  dsn: "__DSN__",
}, sentryReactInit)
```

**Comment by MaxWeisen** [comment]
> 

Currently implementing sentry with `@sentry/capacitor` and `@sentry/vue` but having trouble with `BrowserTracing` 
How would you initialize `BrowserTracing` in this situation?

Example of what I have below

import * as Sentry from '@sentry/capacitor'
import * as SentryVue from '@sentry/vue'

Sentry.init({
  dsn: "__DSN__",
  integrations: [new BrowserTracing()] // <- TS error
}, SentryVue.init)

**Comment by piotr-cz** [comment]
@MaxWeisen
Try pinning `@sentry/vue` to same version as in `@sentry/capacitor`'s [peeerDependencies](https://github.com/getsentry/sentry-capacitor/blob/main/package.json):

```console
npm install --save-exact @sentry/vue@6.19.2
```

For explaination see https://github.com/getsentry/sentry-javascript/issues/4474#issuecomment-1028292131

**Comment by lucas-zimerman** [comment]
Yes, for the time being, sentry/capacitor is not compatible with the newer versions of Sentry/vue, react, angular (7.x).
An update to the SDK will be done after the conclusion of the following update to React Native: https://github.com/getsentry/sentry-react-native/pull/2250

**Comment by davidemerritt** [comment]
Any update on this?
